examples/cpp/pyperf/PyPerfNativeStackTrace.cc

```diff
@@ -65,26 +84,29 @@ NativeStackTrace::NativeStackTrace(uint32_t pid, const unsigned char *raw_stack,
     goto out;
   }
 
+  struct bcc_symbol resolved_symbol;
+  if (NativeStackTrace::insert_dso_name) {
+    procSymbols = get_proc_symbols(pid);
+  }
+
   do {
     unw_word_t offset;
-    char sym[256];
-    char   *realname;
-    int     status;
-
-    // TODO: This function is very heavy. We should try to do some caching here, maybe in the
```

Comment:
Let's keep the TODO next to `unw_get_proc_name`?

Reply:
Sure.